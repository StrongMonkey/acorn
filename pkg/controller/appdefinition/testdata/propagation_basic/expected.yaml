kind: Namespace
apiVersion: v1
metadata:
  name: app-created-namespace
  annotations:
    foo: foo1
  labels:
    bar: bar1
    "acorn.io/app-namespace": "acorn"
    "acorn.io/app-name": "default"
    "acorn.io/managed": "true"
    "pod-security.kubernetes.io/enforce": baseline