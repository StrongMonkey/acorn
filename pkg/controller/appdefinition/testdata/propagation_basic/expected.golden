`apiVersion: v1
kind: Namespace
metadata:
  annotations:
    foo: foo1
  creationTimestamp: null
  labels:
    acorn.io/app-name: default
    acorn.io/app-namespace: acorn
    acorn.io/managed: "true"
    bar: bar1
    pod-security.kubernetes.io/enforce: baseline
  name: app-created-namespace
spec: {}
status: {}
`
